cartesian/.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__
```

Comment:
This file should be deleted - if `__pycache__` needs to be ignored, it can be added to the `.gitignore` file in the root directory.

Reply:
Okay.